docs/client/ui/engine_ui.md

```diff
@@ -0,0 +1,40 @@
+| Share Level    | The share level of engine, such as user, connection, group and server                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| User           | The user created the engine                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| Version        | The version of the Kyuubi server associated with this engine                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| Operation      | Extra operations that users can do further.<br/> 1. View native engine UI<br/>find and select the engine you wish to view its native UI. <br/>clink on the view button, you should be redirected to the native engine UI powered by Kyuubi proxy.  <br/>2. Delete the specified engine gracefully <br/>select the specific engine you would like to delete from the Engine Management page. <br/>click on delete button and confirm your choice, then the engine will be remove from service discovery like Zookeeper, ETCD and etc. <br/>The engine will eventually be shut down once all the jobs running on it inside are finished. |
```

Comment:
> The engine will eventually be shut down once all the jobs running on it inside are finished.

The engine will eventually be shut down once all connected session closed

Reply:
Got it.